cs481-ekh/f21-welolo
# Cypress testing for Mock Merchant page and Item Selection (#44)

This should be merged after #43 

Verification Steps:
- Run cypress
- Run `mock_merchant ` tests
- All tests should be passing
